client/src/pages/Configurations/index.test.tsx

```diff
@@ -10,24 +17,116 @@ const renderPageView = () =>
     </BrowserRouter>
   );
 
-describe('Configurations', () => {
-  it('should contain a table with certain columns', async () => {
-    renderPageView();
+vi.mock('focus-trap-react');
+vi.mock('tabbable');
 
-    expect(screen.getByTestId('table')).toBeInTheDocument();
-  });
-  it('should contain a call-to-action button', async () => {
+describe('Configurations Page', () => {
+  test('renders the Configurations page with title and search bar', () => {
     renderPageView();
     expect(
-      screen.getByText('Set up new condition', {
-        selector: 'button.usa-button[type=button]',
-      })
+      screen.getByText('Your reportable condition configurations')
     ).toBeInTheDocument();
-  });
-  it('should have a search component with correct placeholder', async () => {
-    renderPageView();
     expect(
       screen.getByPlaceholderText('Search configurations')
     ).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: 'Set up new condition' })
+    ).toBeInTheDocument();
+  });
+
+  test('opens the modal when "Set up new condition" button is clicked', () => {
+    const user = userEvent.setup();
+    renderPageView();
+    const setUpButton = screen.getByRole('button', {
+      name: 'Set up new condition',
+    });
+    user.click(setUpButton);
```

Comment:
Functions on the `user` object will return a `Promise` so we'll want to make sure we `await` them so that we don't accidentally introduce any timing issues.
```suggestion
    await user.click(setUpButton);
```